iceberg/iceberg-common/src/main/java/org/apache/gravitino/iceberg/common/ops/IcebergCatalogWrapper.java

```diff
@@ -78,7 +78,9 @@ public class IcebergCatalogWrapper implements AutoCloseable {
           IcebergConstants.ICEBERG_S3_ENDPOINT,
           IcebergConstants.ICEBERG_OSS_ENDPOINT,
           IcebergConstants.ICEBERG_OSS_ACCESS_KEY_ID,
-          IcebergConstants.ICEBERG_OSS_ACCESS_KEY_SECRET);
+          IcebergConstants.ICEBERG_OSS_ACCESS_KEY_SECRET,
```

Comment:
please remove `ICEBERG_ADLS_STORAGE_ACCOUNT_NAME ` and `ICEBERG_ADLS_STORAGE_ACCOUNT_KEY ` , or the client Iceberg will use the above static secret keys.